xchange-core/src/main/java/org/knowm/xchange/dto/trade/UserTrade.java

```diff
@@ -74,6 +77,14 @@ public Currency getFeeCurrency() {
     return feeCurrency;
   }
 
+  public String getOrderUserReference() {
+    return orderUserReference;
+  }
+
+  public void setOrderUserReference(String orderUserReference) {
+    this.orderUserReference = orderUserReference;
+  }
```

Comment:
UserTrade should not be mutable

Reply:
please also make Trade immutable again as I see that its you who added two setters the a year ago

Reply:
OK, i will delete the two setters on Trade.

Reply:
I will completely delete the two maker and taker fields and i will add the orderUserReference to the UserTrade constructor. Is this what you want?